npe2/manifest/schema.py

```diff
@@ -62,12 +62,6 @@ class PluginManifest(BaseModel):
         "package name for this plugin.",
     )
 
-    author: Optional[str] = Field(
```

Comment:
@nclack, please note/comment on the removal of these fields.  With the package_metadata object, they seem redundant.  It's nice to remove them (to single source them), but can think of two potential issues:

- makes it a bit harder to create a manifest not based on a package (i.e. for dynamic creation).  but it's still not immediately clear to me where that will be done.  and we can always add a convenience mocker when we need it.
- makes it a bit harder for the _hub_ to parse this information statically ... since now they need to look in multiple places

Reply:
I think the only one I want to consider keeping is "description". I made a separate comment around that.

> not immediately clear to me where that will be done. and we can always add a convenience mocker when we need it

Totally agree

> harder for the hub to parse [..] statically

I think doing this (relying on pypi) helps the hub maintain clean sources of truth for their metadata.  I'm a big fan of keeping the plugin metadata as bare bones as we can.